### DISCUSSION TITLE
ERROR Highchart-more: Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'addEvent')

### DISCUSSION THREAD

**Discussion by harleyka** [richardeschloss/nuxt-highcharts#69] [General]
Hi, I have the same issue like somebody else here: https://stackoverflow.com/questions/76269680/nuxt-highcharts-more-gives-addevent-of-undefined/77051461#77051461

I needed spiderweb chart, so I used the component:
`<highchart :more="true" :options="chartOptions" />`

But I am getting following error:
`Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'addEvent')
    at highcharts-more.src.js:48:24
    at D (highcharts-more.src.js:26:25)
    at highcharts-more.src.js:38:5
    at highcharts-more.src.js:19:9
    at highcharts-more.src.js:8:2`

Do you know, where the problem is?

**Comment by richardeschloss** [comment]
Hi, please see https://github.com/richardeschloss/nuxt-highcharts/blob/master/components/charts/PackedBubble.vue for a working example of using the "more" property. That example uses the packedBubble chart but I believe you can use the options for spiderweb chart there.

Hmm, https://stackblitz.com/edit/nuxt-starter-5pftra?file=components%2FSpiderChart.vue,app.vue has a repro...hmm.. I'll investigate. (Related discussion: https://github.com/richardeschloss/nuxt-highcharts/discussions/63)

**Comment by richardeschloss** [comment]
Hmm, it seems that the "more" feature is a bit brittle, but the way to use Highcharts more functionality is still achievable:

Working example (Spiderweb): https://stackblitz.com/edit/nuxt-starter-5pftra?file=components%2FSpiderChart.vue

**Comment by harleyka** [comment]
Thanks, it works like this now :)

**Reply by richardeschloss** [reply]
Thanks. I just pushed a fix to the "more" feature in 3.1.1. https://stackblitz.com/edit/nuxt-starter-5pftra?file=package.json,app.vue
